api/abortion_policy_api.py

```diff
@@ -169,9 +141,7 @@ def set_default_types(state_policies):
     for _, state_info in state_policies.items():
         for k, v in state_info.items():
             if k not in keys_and_defaults:
-                key = re.sub(
-                    FILTERED_CHARACTERS_REGEX, "", str(type(v))
-                ).split()[-1]
+                key = re.sub(FILTERED_CHARACTERS_REGEX, "", str(type(v))).split()[-1]
```

Comment:
I think your `black` is running without the `--line-length=80 command for some reason.